tmail-backend/mailbox/encrypted/api/src/main/scala/com/linagora/tmail/encrypted/EncryptedEmailContentStore.scala

```diff
@@ -0,0 +1,66 @@
+package com.linagora.tmail.encrypted
+
+import org.apache.james.blob.api.BlobId
+import org.apache.james.mailbox.model.MessageId
+import org.reactivestreams.Publisher
+import reactor.core.scala.publisher.SMono
+
+object EncryptedEmailContentStore {
+  val POSITION_NUMBER_START_AT: Int = 0
+}
+
+trait EncryptedEmailContentStore {
+  def store(messageId: MessageId, encryptedEmailContent: EncryptedEmailContent)
```

Comment:
Why do we need the non reactive version at all? (can you remove it?)

In other places we have it only to avoid breaking changes and limit the amount of refactoring to be performed, which is not a problem for newly introducted interfaces that IMO would benefit being fully reactive.